incubator/schema-registry/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: schema-registry
 home: https://docs.confluent.io/current/schema-registry/docs/index.html
-version: 1.1.8
```

Comment:
schema reg update is unrelated to PR subject

Reply:
Rebased and pushed; updated to 5.4.1.
reg must've along with Kafka - these containers are built and released together
https://hub.docker.com/r/confluentinc/cp-schema-registry/tags